src/modules/core/components/FeedItem/FeedItem.tsx

```diff
@@ -9,11 +9,11 @@ import TextDecorator from '~lib/TextDecorator';
 import { AnyUser } from '~data/index';
 import FriendlyName from '~core/FriendlyName';
 import { useDataFetcher } from '~utils/hooks';
-import { ipfsDataFetcher } from '../../../../core/fetchers';
+import { ipfsDataFetcher } from '../../fetchers';
 
-import styles from './ActionsPageFeedItem.css';
+import styles from './FeedItem.css';
 
-const displayName = 'dashboard.ActionsPageFeed.ActionsPageFeedItem';
+const displayName = 'FeedItem';
```

Comment:
This should be renamed to `Comment` or `CommentItem` (as the feed name works only in the context of the Actions Page)